util/helper-cairo.hh

```diff
@@ -67,6 +73,21 @@ _cairo_eps_surface_create_for_stream (cairo_write_func_t  write_func,
 #    undef HAS_EPS
 #  endif
 #endif
+#ifdef CAIRO_HAS_SCRIPT_SURFACE
+#   include <cairo-script.h>
+
+static cairo_surface_t *
+_cairo_script_surface_create_for_stream (cairo_write_func_t write_func,
+				         void *closure,
+				         double width,
+				         double height,
+				         cairo_content_t content,
+				         image_protocol_t protocol HB_UNUSED)
+{
+  cairo_device_t *script = cairo_script_create_for_stream (write_func, closure);
```

Comment:
We are leaking the cairo device. No idea where to destroy it, though.

Reply:
Attach it to the surface as user-data. See `png_closure`.

Reply:
I'm on it.

Reply:
I just pushed the change.

Reply:
Actually, just destroying script after creating the surface should do it, no?

Reply:
Right!

Reply:
Are you pushing the simpler version?